uniswap-v2/logics/impls/router/router.rs

```diff
@@ -425,20 +426,22 @@ impl<T: Storage<data::Data>> Router for T {
         Ok(())
     }
 
-    /// Original Uniswap Library pairFor function calculate pair contract address without making cross contract calls.
-    /// Please refer https://github.com/Uniswap/v2-periphery/blob/master/contracts/libraries/UniswapV2Library.sol#L18
-    ///
-    /// In this contract, use cross contract call to get pair contract address.
     default fn _pair_for(
```

Comment:
I would suggest to still leave a doc comment that this is the equivalent of CREATE2 function in solidity + link to substrate where the accountId of the deployed contract is calculated from these args.

Reply:
Added